src/components/Transactions/Transactions.js

```diff
@@ -123,20 +147,19 @@ const Transactions = ({
               }
             )}
           </TransactionListStyled>
-          {!isShowMoreButtonHidden && (
+          {showToggleButton && (
             <Button
               theme="primary"
               margin="20px 0 0 auto"
               width="unset"
-              label={(isExpanded && t('Show less')) || t('Show more')}
+              label={
+                (isTransactionListExpanded && t('Show less')) || t('Show more')
+              }
               onClickFn={() => toggleTransactionsList()}
               padding="12px 33px 12px 20px"
             >
-              <ButtonTextStyled isExpanded={isExpanded}>
-                {(isTransactionsItemsLoading && (
-                  <Loader buttonLoader color="#ffffff" />
-                )) ||
-                  (isExpanded && t('Show less')) ||
+              <ButtonTextStyled isExpanded={isTransactionListExpanded}>
+                {(isTransactionListExpanded && t('Show less')) ||
                   t('Show more')}
```

Comment:
same here: ` isTransactionListExpanded ? t('Show less')) : t('Show more')`

Reply:
I created separate task for refactor TransactionList, but I can delete it and get rid of TransactionList.container as part of this refactor. It seems to me that the only difference between the TransactionList and Transactions components is the header in the TransactionList, but I can create prop for that.